app/core/components/SortInput/index.tsx

```diff
@@ -14,6 +14,11 @@ interface SortOption {
 export const SortInput = ({ setSortQuery, sortBy }: iProps) => {
   //sorting options
   const sortOptions: { [key: string]: SortOption } = {
+    hotness: {
+      label: "Hotness",
```

Comment:
"Hotness" looks a little weird to me, wouldn't be better something like "Hot 🔥"?